packages/web-components/fast-foundation/src/select/select.template.ts

```diff
@@ -46,17 +50,17 @@ export function selectTemplate(
                         ${ref("control")}
                     >
                         ${startSlotTemplate(options)}
-                        <slot name="button-container">
+                        <slot name="control">
```

Comment:
This isn't replacing the entire control so I'm not a fan of this as the name for this slot. This is specifically replacing the invoking mechanism. The entire select in this scenario is the control.